diff -up OpenColorIO-2.1.1/src/OpenColorIO/Config.cpp.1~ OpenColorIO-2.1.1/src/OpenColorIO/Config.cpp
--- OpenColorIO-2.1.1/src/OpenColorIO/Config.cpp.1~	2022-01-24 17:14:42.903777461 +0100
+++ OpenColorIO-2.1.1/src/OpenColorIO/Config.cpp	2022-01-24 17:38:45.002158592 +0100
@@ -1070,12 +1070,6 @@ public:
 
 };
 
-
-
-// Instantiate the cache with the right types.
-extern template class ProcessorCache<std::size_t, ProcessorRcPtr>;
-
-
 ///////////////////////////////////////////////////////////////////////////
 
 ConfigRcPtr Config::Create()
