diff -uraN OpenColorIO-2.0.1/src/libutils/oiiohelpers/CMakeLists.txt omv-OpenColorIO-2.0.1/src/libutils/oiiohelpers/CMakeLists.txt
--- OpenColorIO-2.0.1/src/libutils/oiiohelpers/CMakeLists.txt	2021-05-04 06:36:34.000000000 +0200
+++ omv-OpenColorIO-2.0.1/src/libutils/oiiohelpers/CMakeLists.txt	2021-07-21 02:28:52.344804085 +0200
@@ -35,11 +35,6 @@
         utils::strings
 )
 
-install(TARGETS oiiohelpers
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib
-)
-
 install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/oiiohelpers.h
     DESTINATION include/OpenColorIO/oiiohelpers
 )
diff -uraN OpenColorIO-2.0.1/src/OpenColorIO/CMakeLists.txt omv-OpenColorIO-2.0.1/src/OpenColorIO/CMakeLists.txt
--- OpenColorIO-2.0.1/src/OpenColorIO/CMakeLists.txt	2021-05-04 06:36:34.000000000 +0200
+++ omv-OpenColorIO-2.0.1/src/OpenColorIO/CMakeLists.txt	2021-07-21 02:30:23.198807872 +0200
@@ -279,7 +279,7 @@
 endif()
 
 install(TARGETS OpenColorIO
-	LIBRARY DESTINATION lib
+	LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
 	RUNTIME DESTINATION bin
-	ARCHIVE DESTINATION lib
+	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
 )
