diff -uraN OpenColorIO-2.3.0/share/cmake/modules/Findminizip-ng.cmake omv-OpenColorIO-2.3.0/share/cmake/modules/Findminizip-ng.cmake
--- OpenColorIO-2.3.0/share/cmake/modules/Findminizip-ng.cmake	2023-09-01 03:38:17.000000000 +0200
+++ omv-OpenColorIO-2.3.0/share/cmake/modules/Findminizip-ng.cmake	2023-12-05 18:20:10.608430860 +0100
@@ -102,8 +102,7 @@
 
         # Search for minizip-ng.pc
         find_package(PkgConfig QUIET)
-        pkg_check_modules(PC_minizip-ng QUIET "minizip-ng>=${minizip-ng_FIND_VERSION}")
-
+        pkg_check_modules(PC_minizip-ng QUIET "minizip>=${minizip-ng_FIND_VERSION}")
         # Find include directory
         find_path(minizip-ng_INCLUDE_DIR
             NAMES
@@ -114,6 +113,7 @@
             PATH_SUFFIXES
                 include
                 include/minizip-ng
+                include/minizip
                 minizip-ng/include
         )
