--- ./share/sphinx/ExtractRstFromSourceCPP.py.orig	2014-09-28 08:41:46.530608820 +0100
+++ ./share/sphinx/ExtractRstFromSourceCPP.py	2014-09-28 08:42:08.555607198 +0100
@@ -146,7 +146,7 @@
         
         if buffer_lines[0] != '':
             buffer_lines.insert(0, '')
-        for x in xrange(0, len(buffer_lines)):
+        for x in range(0, len(buffer_lines)):
             buffer_lines[x] = "   %s" % buffer_lines[x]
         buffer_lines.append('')
         buffer = '\n'.join(buffer_lines)
@@ -181,7 +181,7 @@
                     indent += 1
             # remove indent
             bufa = [buf_lines[0]]
-            for x in xrange(1, len(buf_lines)):
+            for x in range(1, len(buf_lines)):
                 bufa.append(buf_lines[x][indent:])
             buf = '\n'.join(bufa) + '\n'
             ##
@@ -229,7 +229,7 @@
             sys.exit(1)
         
         src = open(sys.argv[1]).read()
-        output = file(sys.argv[2], 'w')
+        output = open(sys.argv[2], mode='w')
         ExtractRst(src, output)
         output.close()
         
