diff -up OpenColorIO-2.2.1/src/OpenColorIO/Config.cpp.1~ OpenColorIO-2.2.1/src/OpenColorIO/Config.cpp
--- OpenColorIO-2.2.1/src/OpenColorIO/Config.cpp.1~	2023-01-15 13:03:07.008728948 +0100
+++ OpenColorIO-2.2.1/src/OpenColorIO/Config.cpp	2023-01-15 13:05:31.369604175 +0100
@@ -1504,12 +1504,6 @@ public:
     }
 };
 
-
-
-// Instantiate the cache with the right types.
-extern template class ProcessorCache<std::size_t, ProcessorRcPtr>;
-
-
 ///////////////////////////////////////////////////////////////////////////
 
 ConfigRcPtr Config::Create()
