diff -up OpenColorIO-2.2.1/share/cmake/modules/FindExtPackages.cmake.omv~ OpenColorIO-2.2.1/share/cmake/modules/FindExtPackages.cmake
--- OpenColorIO-2.2.1/share/cmake/modules/FindExtPackages.cmake.omv~	2023-01-15 15:10:39.983116459 +0100
+++ OpenColorIO-2.2.1/share/cmake/modules/FindExtPackages.cmake	2023-01-15 15:10:55.856499436 +0100
@@ -69,7 +69,8 @@ find_package(Imath 3.0 REQUIRED)
 # ZLIB 1.2.13 is used since it fixes a critical vulnerability.
 # See https://nvd.nist.gov/vuln/detail/CVE-2022-37434
 # See https://github.com/madler/zlib/releases/tag/v1.2.13
-set(_ZLIB_FIND_VERSION "1.2.13")
+# zlib-ng is also fixed, but claims to be 1.2.11, so allow it
+set(_ZLIB_FIND_VERSION "1.2.11")
 set(_ZLIB_ExternalProject_VERSION ${_ZLIB_FIND_VERSION})
 
 if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUAL ALL)
diff -up OpenColorIO-2.2.1/share/cmake/modules/Findminizip-ng.cmake.omv~ OpenColorIO-2.2.1/share/cmake/modules/Findminizip-ng.cmake
--- OpenColorIO-2.2.1/share/cmake/modules/Findminizip-ng.cmake.omv~	2023-01-15 15:16:20.444415834 +0100
+++ OpenColorIO-2.2.1/share/cmake/modules/Findminizip-ng.cmake	2023-01-15 15:16:55.381237920 +0100
@@ -94,7 +94,7 @@ if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUA
 
         # Search for minizip-ng.pc
         find_package(PkgConfig QUIET)
-        pkg_check_modules(PC_minizip-ng QUIET "minizip-ng>=${minizip-ng_FIND_VERSION}")
+        pkg_check_modules(PC_minizip-ng QUIET "minizip>=${minizip-ng_FIND_VERSION}")
 
         # Find include directory
         find_path(minizip-ng_INCLUDE_DIR
@@ -106,6 +106,7 @@ if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUA
             PATH_SUFFIXES
                 include
                 include/minizip-ng
+		include/minizip
                 minizip-ng/include
         )
 
diff -up OpenColorIO-2.2.1/src/OpenColorIO/CMakeLists.txt.omv~ OpenColorIO-2.2.1/src/OpenColorIO/CMakeLists.txt
