diff -uraN OpenColorIO-2.2.0/src/OpenColorIO/Config.cpp omv-OpenColorIO-2.2.0/src/OpenColorIO/Config.cpp
--- OpenColorIO-2.2.0/src/OpenColorIO/Config.cpp	2022-10-31 23:27:35.000000000 +0100
+++ omv-OpenColorIO-2.2.0/src/OpenColorIO/Config.cpp	2022-12-21 17:14:58.421444530 +0100
@@ -1505,11 +1505,6 @@
 };
 
 
-
-// Instantiate the cache with the right types.
-extern template class ProcessorCache<std::size_t, ProcessorRcPtr>;
-
-
 ///////////////////////////////////////////////////////////////////////////
 
 ConfigRcPtr Config::Create()
